frontend/src/components/common/header/header.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'common/types/types';
+import { Image } from 'components/common/common';
+import logo from 'assets/img/logo.svg';
+import defaultUserAvatar from 'assets/img/avatar-default.png';
+
+import styles from './styles.module.scss';
+
+const Header: FC = () => (
+  <header>
+    <div className={styles['header-wrapper']}>
+      <div className={styles['logo-wrapper']}>
+        <Image width="150" height="94" src={logo} alt="logo" />
+      </div>
+      <div className={styles['user-wrapper']}>
+        <Image
+          width="50"
+          height="50"
```

Comment:
Can't we pass dimensions for images by styling instead of direct attributes?

Reply:
need to do it on the both sides

Reply:
@what1s1ove @andanfe did not get you guys here, could you please give more details on what should be fixed? thanks

Reply:
i think we fine now 👍